### PR TITLE
Add mysql to nightly

### LINKED ISSUE #21199: [Bug]: Need to add mysql to nightly kafka
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Need to add mysql to nightly kafka

### Expected Behavior

added

### Steps To Reproduce

_No response_

### Milvus Log

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by NicoYuan1986**
Resolve: #21199
Signed-off-by: nico <cheng.yuan@zilliz.com>

### CODE REVIEW

**Comment by binbinlv** [comment]
@NicoYuan1986 
Add to kafka only, not all pipelines. Please modify it. Thanks.

File: ci/jenkins/Nightly.groovy [hunk 1]
```diff
@@ -141,6 +144,8 @@ pipeline {
                                                     --set queryNode.replicas=2 \
                                                     --set indexNode.replicas=2 \
                                                     --set dataNode.replicas=2 \
+                                                    --set mysql.enabled=true \
```
Comment (binbinlv) [inline ci/jenkins/Nightly.groovy:147]: Add to kafka only, not all pipelines. Please modify it. Thanks.

Comment (NicoYuan1986) [reply ci/jenkins/Nightly.groovy]: > Add to kafka only, not all pipelines. Please modify it. Thanks.

Modified.

**Pull request merged**